apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/springsecurity/SpringSecurityUserService.java

```diff
@@ -91,7 +92,7 @@ public List<UserInfo> searchUsers(String keyword, int offset, int limit) {
 
   private List<UserPO> findUsers(String keyword) {
     if (StringUtils.isEmpty(keyword)) {
-      return userRepository.findFirst20ByEnabled(1);
+      return (List<UserPO>) userRepository.findAll();
```

Comment:
We can't change this logic directly as it is also used elsewhere like permission management. With this change, the disabled users will be shown in permission management, which is not expected.
I think maybe we could add a parameter `includeInactiveUsers` to UserService.searchUsers, e.g.

```java
List<UserInfo> searchUsers(String keyword, int offset, int limit, boolean includeInactiveUsers);
```

Then we need to also add a parameter to UserInfoController.searchUsersByKeyword, e.g.

```java
  @GetMapping("/users")
  public List<UserInfo> searchUsersByKeyword(@RequestParam(value = "keyword") String keyword,
      @RequestParam(value = "includeInactiveUsers", defaultValue = "false") boolean includeInactiveUsers,
      @RequestParam(value = "offset", defaultValue = "0") int offset,
      @RequestParam(value = "limit", defaultValue = "10") int limit) {
    return userService.searchUsers(keyword, offset, limit, includeInactiveUsers);
  }
```